unpackagecsharp.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```

Comment:
this file shouldn't be in the root of cozmo-misc-sdk, and ideally would be in another repo entirely as it's not intended / required for use by end users so needn't be in a public repo